assets/js/base/components/product-price/index.js

```diff
@@ -119,12 +119,26 @@ const ProductPrice = ( {
 		}
 	);
 
+	if ( ! format.includes( '<price/>' ) ) {
+		format = '<price/>';
+		// eslint-disable-next-line no-console
+		console.error( 'Price formats need to include the `<price/>` tag.' );
```

Comment:
I'm planning to improve this and other protections checks in #3735.

Reply:
This should now be removed?